sdk/python/feast/type_map.py

```diff
@@ -430,6 +432,50 @@ def redshift_to_feast_value_type(redshift_type_as_str: str) -> ValueType:
 
     return type_map[redshift_type_as_str.lower()]
 
+def snowflake_native_type_to_feast_value_type(snowflake_native_type_as_str: str) -> ValueType:
```

Comment:
revert these since they're not used

Reply:
snowflake_python_type_to_feast_value_type is used, but deleted other